engine/language-server/src/main/scala/org/enso/languageserver/boot/resource/RepoInitialization.scala

```diff
@@ -34,10 +36,25 @@ class RepoInitialization(
   /** @inheritdoc */
   override def init(): Future[InitializationComponent.Initialized.type] =
     for {
+      _ <- sqlDatabaseInit
```

Comment:
Does this run in its own `Thread`? Or does it run sequentially first initializaing `sqlDatabaseInit` then `suggestionRepoInit` and then `versionRepoInit`?

Reply:
> Does this run in its own `Thread`? Or does it run sequentially first initializaing `sqlDatabaseInit` then `suggestionRepoInit` and then `versionRepoInit`?

The former.

Reply:
Yes, the actions are sequential. Later I'll review which parts of initialization can run in parallel